pkgs/development/ocaml-modules/ffmpeg/base.nix

```diff
@@ -1,13 +1,13 @@
 { lib, fetchFromGitHub }:
 
 rec {
-  version = "1.1.8";
+  version = "1.1.11";
 
   src = fetchFromGitHub {
     owner = "savonet";
     repo = "ocaml-ffmpeg";
-    rev = "v${version}";
-    sha256 = "sha256-XqZATaxpW0lEdrRTXVTc0laQAx437+eoa/zOzZV1kHk=";
+    rev = "refs/tags/v${version}";
```

Comment:
Why do we need this suddenly? Is there a branch with the same name?

Reply:
common hardening found elsewhere in nixpks

Reply:
> Is there a branch with the same name?

Yes